### DISCUSSION TITLE
Invalid choice 'db' after updating to Dropwizard 4.0.0

### DISCUSSION THREAD

**Discussion by harpreet86** [dropwizard/dropwizard#7234] [Q&A] [answered]
We recently migrated our project from 2.1.6 to 4.0.0

After this, we are getting the below error while running db migrate, please suggest:

```

invalid choice: 'db' (choose from 'server', 'check')

usage: java -jar project.jar [-h] [-v] {server,check} ...

 

positional arguments:

  {server,check}         available commands

 
named arguments:

  -h, --help             show this help message and exit

  -v, --version          show the application version and exit
```

**Comment by joschi** [comment, marked as answer]
The `db` command is contributed by `dropwizard-migrations`: https://www.dropwizard.io/en/stable/manual/migrations.html

Did you properly add the `dropwizard-migrations` bundle (in the correct version) to your application?

**Reply by harpreet86** [reply]
I have tried with maven dependency 

```
<dependency>
    <groupId>io.dropwizard</groupId>
    <artifactId>dropwizard-migrations</artifactId>
    <version>4.0.0</version>
</dependency>
```
but still facing the same issue.

**Reply by joschi** [reply]
Please post the code of your `Application` class where you register the bundle.
